Utils/Elasticsearch/tools/daily/es_backup.logrotate

```diff
@@ -0,0 +1,46 @@
+nocreate
+nocompress
+nomail
+missingok
+dateext
+extension .tar.gz
+
+/var/es/backup/data.tar.gz {
+    daily
+    rotate 7
+    prerotate
+        base_dir=/var/es/backup
+        # create dirs for weekly/monthly/yearly copies
+        for d in weekly monthly yearly; do
+            mkdir -p -m 0700 "${base_dir}/$d"
+        done
+
+        # copy rotated file:
+        # - to weekly backups every Sunday
+         [ $(date "+%u") -eq 7 ] && ln "$1" "$base_dir"/weekly/"`basename $1`"
```

Comment:
There is not needed extra quotes in the middle of the string _"$base_dir"/weekly/"\`basename $1\`"_

Reply:
Fixed: a2633ba53454f5bbaf213f5b0e84e6fc2929767b